### ISSUE TITLE
Padded AES

### ISSUE THREAD

**Issue by godylockz** [gchq/CyberChef#1322]
For AES Encrypt/Decrypt, I would like to enter a key that is UTF-8 for instance and is shorter than the required key length and it is padded with 0s (NULLs) 

There are tools that currently do this however CyberChef does not.
This could potentially be a checkbox on AES Decrypt/Encrypt.

**Comment by KagamigawaMeguri** [comment]
Agree. When i use AES Decrypt, some character is not parsed properly, but other tools that has zeropadding options is ok.
